scanpipe/models.py

```diff
@@ -1248,6 +1257,9 @@ def has_licenses(self):
     def has_no_licenses(self):
         return self.filter(licenses=[])
 
+    def has_package_data(self):
+        return self.filter(package_data__isnull=False)
```

Comment:
`isnull` is not the proper lookup to use for this field that has an empty list a default value.

```
self.filter(~Q(package_data=[]))
```